Active Active expiration values (#1475)
RedisLabs/redislabs-docs: content/rs/references/developing-for-active-active/_index.md

    DEL.
- Expires it (sending a DEL) before making any modifications if a user
    attempts to access it in WRITE mode.
    
    {{< note >}}

> Active-Active databases expiration values are in range of [0, 2^49].
> While non Active-Active Redis expiration values are in range of [0, 2^64], 

Reviewer: ```suggestion
Expiration values are in the range of [0, 2^49] for Active-Active databases and [0, 2^64] for non Active-Active databases.
```
